images/eh-expansion-evaluator/Dockerfile

```
@@ -0,0 +1,17 @@
+FROM ubuntu:20.04
```

This is pretty old by now.
```suggestion
FROM ubuntu:22.04
```